docs/deploying-airbyte/on-oci-vm.md

```diff
@@ -63,34 +101,47 @@ wget https://raw.githubusercontent.com/airbytehq/airbyte/master/{.env,docker-com
 
 which docker-compose
 
-docker-compose up -d
+sudo /usr/local/bin/docker-compose up -d
 ```
 
+### Airbyte URL Access Method 1 : Local Port Forward to Airbyte VM using Bastion host
 
-## Create SSH Tunnel to Login to the Instance
+```text
+ssh opc@bastion-host-public-ip -i <private-key-file.key> -L 8000:oci-private-instance-ip:8000
+```
 
-it is highly recommended to not have a Public IP for the Instance where you are running Airbyte\).
+### Access Airbyte
+
+Open URL in Browser : [http://localhost:8000/](http://localhost:8000/)
 
-### SSH Local Port Forward to Airbyte VM
+![](../.gitbook/assets/OCIScreen4.png)
```

Comment:
Have you added this file? Or does it already exist?

Reply:
The older image files already exist. How can i add the new screenshots